benchmarks/replay/replay.cpp

```diff
@@ -128,16 +151,16 @@ struct replay_benchmark {
   replay_benchmark(replay_benchmark const&) = delete;
 
   /// Add an allocation to the map (NOT thread safe)
-  void set_allocation(std::unordered_map<uintptr_t, allocation>& allocation_map,
-                      uintptr_t ptr,
-                      allocation alloc)
+  static void set_allocation(std::unordered_map<uintptr_t, allocation>& allocation_map,
```

Comment:
I now don't remember why this needed to take allocation_map as an argument. Could it be left non-static and that argument removed?

Reply:
Done.